.github/ISSUE_TEMPLATE.md

```diff
@@ -45,6 +47,31 @@ $ dpkg -l | grep centreon
 
 ***Additional environment details (AWS, VirtualBox, physical, etc.):***
 
+<h3> Component </h3>
```

Comment:
indeed, but code issue maybe linked to a module. And it may help us understand/correct the behavior easier/faster

Reply:
@ganoze That is something that we have discussed with @lpinsivy.
But in fact, now that you raised that question, it will be difficult to handle PR for modules in Centreon repository...

It makes ticket management easier (avoid **duplication**) but for PR it is **way more complicated**.

I'll **remove this section**.

Thanks